### DISCUSSION TITLE
Is there any way to export DTS 7.1 audio from DTS HDMA 7.1 or TrueHD 7.1 ?

### DISCUSSION THREAD

**Discussion by AdithyanNellickal** [staxrip/staxrip#913] [Q&A]
Hi guys,
Is there any way to export DTS 7.1 Audio from DTS-HD MA 7.1 or TrueHD 7.1 while encoding in Stax ? When I tried ORIGINAL CHANNELS option, it DownScaled to 5.1 and when I explicitly mentioned 7.1, it gave me error. So can you guys plz tell me if it's possible to do DTS 7.1 ? 

NB: I'm not asking about DTS Core, but DTS with custom bitrate like 768 etc.

**Comment by jessielw** [comment]
You can encode DTS with the ffmpeg experimental encoder. However, it's limited to mono/stereo output. Although you can control the bitrate.

What you want to do is not possible with free encoders.

I'd suggest encoding with Opus 7.1 as an alternative. Much better vs the DTS core.

**Reply by AdithyanNellickal** [reply]
> You can encode DTS with the ffmpeg experimental encoder. However, it's limited to mono/stereo output. Although you can control the bitrate.
> 
> What you want to do is not possible with free encoders.
> 
> I'd suggest encoding with Opus 7.1 as an alternative. Much better vs the DTS core.

Thanks a lot for the reply bro. When I tried with custom bitrate, I got DTS 5.1 (@768kbps). I just wanted to check if 7.1 channel was possible. 

(For Opus, some of my users will get mad bro if I use Opus in 7-10GB files 🙈🙈🙈)

**Comment by jessielw** [comment]
AC3@640k will destroy DTS 5.1@768k and have significantly higher compatibility. If you're not at least using the DTS core ~1509k, then you might as well use AC3 or Opus. There is arguments that even the DTS core ~1509k is indiscernible from AC3 5.1 640k. DTS is a lossy compression codec, it's only lossless and higher quality when left in it's original format. Once you encode it (especially with the FFMPEG's DTS encoder) you're just wasting space with less quality than other lossy formats. 

Best lossy compression formats that should be used:
-----------------------------------------------------------------------------------------------

**Multi channel formats (quality from least to greatest)**
Opus > AAC > AC3 > all else
**Compatibility with multi channel formats (more than 2.0)**
AC3 > Opus > AAC

**For stereo channels (quality from least to greatest)**
Opus > AAC > AC3
**Compatibility with stereo**
AC3 > AAC > Opus


**The take away, if you're compressing a lossless format (TrueHD / DTS-HD MA / FLAC / or some other lesser used formats) then you should be doing it in AC3, Opus, or AAC depending on the channel layout.** 

_Depends on your use case, if it's for plex encoding audio takes less than 1% CPU power on a modern CPU. So then it just comes to storage, because plex can transcode. If you want it to direct play or for compatibility, you should follow the basic charts up there._

**Reply by AdithyanNellickal** [reply]
I was actually doing EAC3 7.1 @ 768 bro. But some users asked if it's possible to do DTS 7.1 as well, and that's why I was searching. 


And no, I'm not asking for Plex. I have my own movie website (non-profit 😅😅, no ads and all). And I upload my encodes there.

**Reply by jessielw** [reply]
E-AC3 has decreased compatibility. If I was streaming from a website you'd want the most compatibility possible. Which would be AAC 2.0

**Reply by AdithyanNellickal** [reply]
Mine's not a streaming site bro. It's a downloading one.

**Reply by jessielw** [reply]
Well I think I answered this in full

**Reply by sheik124** [reply]
Again, no, you can't do it with any free software, and you certainly can't do it with StaxRip. You'd need a proprietary DTS software encoder.

The people asking you for "DTS 7.1" have no idea what they're talking about and can safely be ignored because it does not exist. DTS (DCA - Digital Coherent Acoustics - the name of the vanilla DTS codec) **does not support 7.1 channel audio**. It's limited to either matrixed or discrete 6.1 channel audio a.k.a. DTS-ES. You would have to downmix it to 6.1, and the DTS core stream embedded in your DTS-HD MA 7.1 track may actually already be DTS-ES flagged. 

>NB: I'm not asking about DTS Core, but DTS with custom bitrate like 768 etc.

That is still "DTS Core". "Core" just refers to a regular DCA stream when it's contained within a DTS-HD MA or DTS-HD HR stream for backwards compatibility. DTS-HD MA and DTS-HD HR only exist as extensions on top of a backwards compatible DCA stream, just like DTS-ES and DTS 96/24 before them.

You would have to use DTS-HR for 7.1, which really has no place outside of Blu-Ray releases that don't have lossless audio, because it's still burdened with a 1.5 Mbps DTS 5.1 or DTS-ES 6.1 core substream. I've only ever seen it used for secondary audio tracks on European Blu-Rays that were in a foreign language but maintained the 7.1 mix. If the codec restrictions of Blu-Ray are not an issue, _why would you ever willingly use it?_

![image](https://user-images.githubusercontent.com/2312934/182040875-e64eb50c-ad3c-411e-afed-95106929eb1d.png)

E-AC-3 7.1 @ 768 is wasteful. DTS 7.1, were it even possible, would be even more wasteful, but DTS-HD HR 7.1 is still wasteful. DCA was only beneficial on DVDs where it was the best option available at 1509 kbps vs. lower bitrate AC-3, in theaters vs. much lower bitrate AC-3 and before DCPs all switched to lossless spatial audio, or for some rare 5.1 music that was only released as a DTS CD. It is a dinosaur. Willingly encoding a lossless source to either format when you don't have hardware/format restrictions limiting you is baffling.

Use `qaac --tvbr 100` if you're paranoid or libopus at your bitrate of choice like you're being advised to...they are lower bitrate than either DD+ or DTS and sound better.

**Reply by jessielw** [reply]
Couldn't have said it better myself and I learned even more. Thanks!

**Reply by AdithyanNellickal** [reply]
> Again, no, you can't do it with any free software, and you certainly can't do it with StaxRip. You'd need a proprietary DTS software encoder.
> 
> The people asking you for "DTS 7.1" have no idea what they're talking about and can safely be ignored because it does not exist. DTS (DCA - Digital Coherent Acoustics - the name of the vanilla DTS codec) **does not support 7.1 channel audio**. It's limited to either matrixed or discrete 6.1 channel audio a.k.a. DTS-ES. You would have to downmix it to 6.1, and the DTS core stream embedded in your DTS-HD MA 7.1 track may actually already be DTS-ES flagged.
> 
> > NB: I'm not asking about DTS Core, but DTS with custom bitrate like 768 etc.
> 
> That is still "DTS Core". "Core" just refers to a regular DCA stream when it's contained within a DTS-HD MA or DTS-HD HR stream for backwards compatibility. DTS-HD MA and DTS-HD HR only exist as extensions on top of a backwards compatible DCA stream, just like DTS-ES and DTS 96/24 before them.
> 
> You would have to use DTS-HR for 7.1, which really has no place outside of Blu-Ray releases that don't have lossless audio, because it's still burdened with a 1.5 Mbps DTS 5.1 or DTS-ES 6.1 core substream. I've only ever seen it used for secondary audio tracks on European Blu-Rays that were in a foreign language but maintained the 7.1 mix. If the codec restrictions of Blu-Ray are not an issue, _why would you ever willingly use it?_
> 
> ![image](https://user-images.githubusercontent.com/2312934/182040875-e64eb50c-ad3c-411e-afed-95106929eb1d.png)
> 
> E-AC-3 7.1 @ 768 is wasteful. DTS 7.1, were it even possible, would be even more wasteful, but DTS-HD HR 7.1 is still wasteful. DCA was only beneficial on DVDs where it was the best option available at 1509 kbps vs. lower bitrate AC-3, in theaters vs. much lower bitrate AC-3 and before DCPs all switched to lossless spatial audio, or for some rare 5.1 music that was only released as a DTS CD. It is a dinosaur. Willingly encoding a lossless source to either format when you don't have hardware/format restrictions limiting you is baffling.
> 
> Use `qaac --tvbr 100` if you're paranoid or libopus at your bitrate of choice like you're being advised to...they are lower bitrate than either DD+ or DTS and sound better.

Thanks for explaining everything bro. 😊😊
